corehq/apps/reports/__init__.py

```diff
@@ -9,4 +9,4 @@
 _("Inspect Data")
 _("Raw Data")
 _("Manage Deployments")
-_("Commtrack")
+_("CommTrack")
```

Comment:
Yes and no - this is just here to be marked for translation, and my thoughts were that if this string IS found, it should be translated to "CommCare Supply".  I've added a comment to clarify.